### PR TITLE
Add Cryptocurrency types

### PR DESCRIPTION
**Pull request by jayadebaj**
Added Cryptocurrency types

### CODE REVIEW

File: v1/schema/json/openapi-2.0/cryptocurrency_symbol.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            "value": "BCH",
+            "description": "The ticker symbol for <b>Bitcoin Cash</b>. https://en.wikipedia.org/wiki/Bitcoin_Cash."
+        },
+        {
+            "value": "LTC",
+            "description": "The ticker symbol for <b>Litecoin</b>. https://en.wikipedia.org/wiki/Litecoin."
+        },
+        {
+            "value": "PYPC",
+            "description": "The ticker symbol for <b>PayPal Coin</b>. https://engineering.paypalcorp.com/confluence/display/BCDC/PayPal+Digital+Coin+%28PPDC%29+-+USDP+-+Top+Down+View+of+Requirements."
```
Comment (mimckenna) [inline v1/schema/json/openapi-2.0/cryptocurrency_symbol.json:26]: Remove internal URL, or replace with https://newsroom.paypal-corp.com/2022-02-03-PayPal-welcomes-leading-experts-in-distributed-technology-economics-and-regulation-to-Blockchain-Crypto-and-Digital-Currencies-advisory-council

**Review by mimckenna — CHANGES_REQUESTED** [review]
Minor change - fix URL for description of PYPC

File: v1/schema/json/openapi-2.0/cryptocurrency_symbol.json (hunk 1 above)
Comment (jayadebaj) [reply v1/schema/json/openapi-2.0/cryptocurrency_symbol.json]: Removed this entry now. We'll add it later.

**Review by mimckenna — APPROVED** [review]
Looks good!

**Pull request merged**